build.sbt

```diff
@@ -228,9 +229,11 @@ lazy val tests = project
         "; tests/it:testQuick" // hack to workaround cyclic dependencies in test.
     )(state.value),
     parallelExecution in Test := true,
+    resolvers += Resolver.bintrayRepo("allanrenucci", "maven"),
```

Comment:
The plan is to have an official release when it gets merged into scala-xml. cc/ @densh